internal/cmd/build/build.go

```diff
@@ -400,4 +449,7 @@ func init() {
 	Cmd.Flags().StringVarP(&outFn, "output", "o", "", "Output to a file")
 	Cmd.Flags().BoolVar(&pklClass, "pkl-class", false, "Output a pkl class based on a resource type schema")
 	Cmd.Flags().BoolVar(&noCache, "no-cache", false, "Do not used cached schema files")
+	Cmd.Flags().BoolVar(&guard, "guard", false, "When using --prompt, output a CloudFormation Guard policy file")
+	Cmd.Flags().BoolVar(&rego, "rego", false, "When using --prompt, output an OPA Rego policy file")
+	Cmd.Flags().StringVar(&model, "model", "claude2", "The ID of the Bedrock model to use for --prompt. Shorthand: claude2, claude3haiku, claude3sonnet")
```

Comment:
just sharing an idea to make this a bit for flexible and easier to maintain let me know what you think:

maybe it would be cleaner to have two options, `--model` which is just a shorthand with fixed set of values like in the description here and another option `--model-id` where users would specify the specific id if they wish to use any other model other than the default one. These two flags would work together ie. the `--model-id` would be dependent on `--model` this way we can avoid the check functions `isClaude2()` because the user will specify the family if they wish to use a different model. An other benefit i see here is that it will be easier to add other family of models in the future. 

TLDR; we will have a clearer input from the user by using `--model` and `--model-id`.

Reply:
I thought about that, but since each model requires a very different input, it would do no good to allow a user to specify an arbitrary model id. Maybe when we add more models, or if there are different model versions with subtle differences in behavior, but no difference in input.

Reply:
understood, I assumed there are variants of claude-2 and claude-3 in bedrock because of the option to enter an id like `anthropic.claude-3-sonnet-20240229-v1:0` instead of the shorthand, this is why i thought it would be nice to let the user specify it like for instance `--model claude3sonnet --model-id anthropic.claude-3-sonnet-20240229-v1:0` or some other claude3sonnet model version. its fine to keep it simple for now :+1: